docs/topics/http/urls.txt

```diff
@@ -682,8 +687,11 @@ the fully qualified name into parts and then tries the following lookup:
       :class:`~django.template.RequestContext` that is used to render a
       template.
 
-   The current application can also be specified manually as an argument
-   to the :func:`~django.core.urlresolvers.reverse` function.
+   .. versionchanged:: 1.9
+
+      In previous versions of Django, the :ttag:`url` template tag did not use
+      the namespace of the currently resolved view, and you had to explicitly
```

Comment:
Also, farther down on this page there's a mention about setting `self.request.current_app` in class based views which can probably be removed.